Cargo.lock

```diff
@@ -10385,7 +10230,7 @@ dependencies = [
  "dirs-next",
  "hex",
  "move-core-types",
- "num-bigint 0.4.4",
+ "num-bigint 0.3.3",
```

Comment:
Here's another downgrade, but apparently due to rand dependency.

Reply:
Same story, that is the workspace version at main: https://github.com/aptos-labs/aptos-core/blob/77e1d222ebc5e7294e115e0d090c001da1d0e072/Cargo.toml#L599